tools/integration_tests/read_large_files/read_large_files_test.go

```diff
@@ -34,10 +36,48 @@ const NumberOfRandomReadCalls = 200
 const MinReadableByteFromFile = 0
 const MaxReadableByteFromFile = 500 * OneMB
 
+func createMountConfigsAndEquivalentFlags() (flags [][]string) {
+	cacheLocationPath := path.Join(os.Getenv("HOME"), "cache-dri")
+
+	// Set up config file for file cache with download-file-for-random-read: false
+	mountConfig1 := config.MountConfig{
+		FileCacheConfig: config.FileCacheConfig{
+			// Keeping the size as high because the operations are performed on large
+			// files
+			MaxSizeInMB:               700,
+			DownloadFileForRandomRead: true,
+		},
+		CacheLocation: config.CacheLocation(cacheLocationPath),
+		LogConfig: config.LogConfig{
+			Severity: config.TRACE,
+		},
+	}
+	filePath1 := setup.YAMLConfigFile(mountConfig1, "config1.yaml")
```

Comment:
Just wondering, do we delete these config files after test?

Reply:
No, we don't.